lib/go-tc/servers.go

```diff
@@ -1019,6 +1019,12 @@ func UpdateServerPropertiesV40(profileNames []string, properties CommonServerPro
 	}
 }
 
+// ServerV41 is the representation of a Server in version 4.1 of the Traffic Ops API.
+type ServerV41 struct {
+	ServerV40
+	ASNs []int64 `json:"asns"`
```

Comment:
The ASNs are defined in postgres as `BIGINT`s. I wanted to preserve the same behavior here.

Reply:
Umm.. where, I checked before writing the above comment. Below is ASN table:
<img width="761" alt="image" src="https://user-images.githubusercontent.com/22248619/186040501-d82418b7-eb3b-4fb8-90c1-08d55039ab99.png">

Reply:
Ah I was looking at the `id`. However, I'd like to keep this as an `int64` because it is one of the supported types in `pq.Scanner`. If I keep this field as an int, I'd have to implement those methods for the type.

Reply:
Per our conversation, I'm ok with resolving the conversation and leaving it at int64